list_test.go

```diff
@@ -209,11 +209,10 @@ func TestErrors(t *testing.T) {
 		{
 			func() (List, bool) { return ParseList("@example.com, invalid") },
 			true,
-			"2 errors occurred:",
+			"1 error occurred:",
 			&multierror.Error{
 				Errors: []error{
 					errors.New("unable to find an email address"),
-					errors.New("unable to find an email address"),
```

Comment:
I had to modify this test, as none of the items on the list are valid addresses but now that unquoted comma is a thing on names we can treat this as 2 items but one

I thing the impact of this will be minimal since only affects errors

Reply:
The use case for the errors to work like this is that you can mark individual addresses as invalid/broken when you type something like `valid@example.com, nope, other@example.com`. You can highlight just the `nope`, which is a nice UI detail.

Coincidently, this is also how I used this recently, because I had a long list of addresses collected from a crappy script I wrote in a hurry and wanted to know which one were invalid.

What kind of software sends `Support <support@email.com <mailto:support@email.com>>` btw :-/ Does any email system work with that?

Reply:
We have the customer on the task which is sending
`Martin, Nichole <Nichole.Martin@a.com <mailto:Nichole.Martin@b.com>>`
It has the nested address, and unquoted name with comma, so bad

About the nested address, I've been looking but couldn't find any documentation about it, but I remember reading once that it is used by some email clients when you are sending a email from an email address that is not your email address.

It used to be used when the email address you are using is from an org but you want to let everybody know who you are and where they should reply
I do not remember it correctly but it was something like that, and a correct usage would be:
`Gerard Rodes <support@org.com <mailto:gerard.rodes@org.com>`
And maybe even 
`Gerard Rodes <support@org.com <mailto:gerard.rodes@gmail.com>`

but again, I reed about it time ago and don't know for sure
If we could find docs about it would be great

Reply:
I'd start by checking the database to see how often this occurs; when I wrote this I tested it on a corpus of ~100k email addresses extracted from Desk and my personal archives, and while I of course didn't manually check every last fault, I did spend a long time looking at the failed parsers especially. I don't recall encountering this.

I wouldn't be surprised if this is just a single broken webform or something. Like I said in the other PR, pragmatism beats purity, but this really seems "too broken" to fix. Before you know it you've accumulated loads of fixes for broken bespoke email scripts that are hardly used and you've ended up with some Sendmail-like monstrosity 🙃

Reply:
I would be interested in getting those emails
Although we should test it against the raw eml, because there is where that syntax is found and I've seen that previously it was ignored.
Like this https://github.com/Teamwork/twmail/pull/17/files#diff-27a1f62996b496bc3bafef83ab67db3f047a2e8e1b7eaa6051229ee8c2120962R200-R220

I had to extend the tests because some forwarding metadata was not taken in account

Reply:
I would be interested in getting those emails
Although we should test it against the raw eml, because there is where that syntax is found, and I've seen that previously it was ignored.

I had to extend the tests because some forwarding metadata was not taken in account

Reply:
This was like five years ago; I don't have any of this data readily available any more. I checked my personal email with a quick grep, and I can't find any examples of this.

I'd be surprised if any major email system would work with an address like that. This seems like a classic case of someone messing up a PHP webform or some such.

Reply:
Yeah it does sound more like a HTML entry error now that you mention it @arp242, especially considering the `mailto` part which is generally only used in HTML as far as I'm aware, to turn the URL into a link to send an email. I can't really imagine that being used in an email itself.

If we could find some examples of it in the wild that'd be good. I haven't personally seen this occur before though.

Reply:
Yeah, you must be right, i am not finding anything like what i am trying to parse here
https://www.google.com/search?q=filetype:eml+intext:mailto+intext:%22%3E%3E%22

Reply:
I'd be in support of not changing this then as it's wide-spanning and dangerous and it's possible it's not coming from actual emails